apps/css-workshop/src/pages/icon.astro

```diff
@@ -53,74 +51,40 @@ import Layout from './_layout.astro';
 
   <h2>Icons with padding</h2>
   <section id='demo-padded'>
-    <svg-imodel-hollow
-      class='iui-svg-icon'
-      data-iui-icon-size='s'
-      data-iui-padded='true'
-      aria-hidden='true'></svg-imodel-hollow>
-    <svg-imodel-hollow
-      class='iui-svg-icon'
-      data-iui-icon-size='m'
-      data-iui-padded='true'
-      aria-hidden='true'></svg-imodel-hollow>
-    <svg-imodel-hollow
-      class='iui-svg-icon'
-      data-iui-icon-size='l'
-      data-iui-padded='true'
-      aria-hidden='true'></svg-imodel-hollow>
+    <Icon_ svg='imodel-hollow' size='s' data-iui-padded='true' />
```

Comment:
The `data-iui-padded` attribute should be handled within `Icon` using a boolean `padded` prop

Reply:
made that switch in last push